modules/experimental/src/index.js

```diff
@@ -4,11 +4,13 @@ export {writeFile} from './core-addons/write-file-browser';
 export {default as AsyncQueue} from './javascript-utils/async-queue';
```

Comment:
Can we just drop the utilities from `experimental` and move them into `core` or `loader-utils`?

Reply:
> Can we just drop the utilities from experimental and move them into core or loader-utils?

Yes the goal is to get all table related stuff out of experimental. I am thinking that we'll need a table category module for all the table batch / arrow compatibility stuff.

The core module is growing really fast right now and I want to be careful with adding things there.

Reply:
These were moved into the `tables` category in previous diffs.